[javascript] Array WebAPIs

sbjang123456 opened the pull request.

SooJungChae reviewed (approved):
앗 split 이 배열사이즈를 리턴하는군요... 
감사합니다. 정리를 너무 잘하십니다!!

SooJungChae commented on javascript/array_webAPIs.md:
````diff
@@ -0,0 +1,122 @@
+const result = people.reduce((prev, curr) => prev + curr.age, 0); // result : 96
+```
+
+> `redueRight` : 배열의 제일 뒤에서 부터 시작
````
이건 어떻게사용하나요??

sbjang123456 replied:
reduce 를 배열의 시작이 아닌 마지막부터 시작한다고하는데, 저도 한번도 써본적은 없네요..
누적을 마지막부터 진행한다고 하던데, 사용방법은 reduce 를 똑같더라구요. (배열 reverse 후 reduce 느낌이 아닐까합니다)

SooJungChae replied:
이런걸 사용하는 개발자는 얼마나 똑똑할까 ...

sbjang123456 commented:
> 앗 split 이 배열사이즈를 리턴하는군요... 감사합니다. 정리를 너무 잘하십니다!!

앗!! 아뇨! 자르고 넘겨받을 배열의 리미트를 매개변수를 넘겨준다는 뜻이었습니다!
```js
const str = 'a,b,c,d,e';
const result1 = strNumber.split(','); // result1 : ['a', 'b', 'c', 'd', 'e']
const result2 = strNumber.split(',', 3); // result2 : ['a', 'b', 'c']
```
